sklearn/_loss/_loss.pyx.tp

```diff
@@ -870,7 +870,9 @@ cdef class CyLossFunction:
         floating_out[::1] loss_out,             # OUT
         int n_threads=1
     ):
-        """Compute the pointwise loss value for each input.
+        """Compute the point-wise loss value for each input.
+
+        The point-wise loss is affected to `loss_out` without returning the array.
```

Comment:
```suggestion
        The point-wise loss is written to `loss_out` and no array is returned.
```
WDYT?

Reply:
Yes much better. I used the french way of saying. This is terrible. I will modify the other occurrences.